reconcile/terraform_resources.py

```diff
@@ -541,20 +548,20 @@ def setup(
     thread_pool_size: int,
     internal: str,
     use_jump_host: bool,
-    account_name: Optional[str],
+    account_names: Optional[str],
 ) -> tuple[ResourceInventory, OC_Map, Terraform, ExternalResourceSpecInventory]:
     accounts = queries.get_aws_accounts(terraform_state=True)
-    if account_name:
-        accounts = [n for n in accounts if n["name"] == account_name]
+    if account_names:
+        accounts = [n for n in accounts if n["name"] in account_names]
         if not accounts:
-            raise ValueError(f"aws account {account_name} is not found")
+            raise ValueError(f"aws accounts {account_names} where not found")
```

Comment:
I think that if one account passed is missing in `accounts` it can because of an typo or something. So we could calculate the diff and let the user know what account is missing in `accounts` and then fail safely.

Reply:
Makes sense!